uefi/CHANGELOG.md

```diff
@@ -15,9 +15,9 @@ details of the deprecated items that were removed in this release.
 - **Breaking:** `FileSystem` no longer has a lifetime parameter, and the
   deprecated conversion from `uefi::table::boot::ScopedProtocol` has been
   removed.
+- **Breaking:** Removed `BootPolicyError` as `BootPolicy`
 - Fixed `boot::open_protocol` to properly handle a null interface pointer.
 
-
```

Comment:
tiny nit: per your changes in https://github.com/rust-osdev/uefi-rs/commit/4d1906e0a03432cac0a42a12c5e978e0dc807388, two blank lines between each version entry